libROOT/libROOT/CMakeLists.txt

```diff
@@ -1,4 +1,13 @@
+include(GNUInstallDirs)
+
 add_library(libROOT SHARED root.cpp
         dummy_class.cpp)
 
 target_include_directories(libROOT PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+
+set_target_properties(libROOT PROPERTIES PREFIX "")
```

Comment:
CMake automatically prefixes libraries with `lib`, so out library was being saved as `liblibROOT`; this fixes that.